src/CoreShop/Bundle/LocaleBundle/Form/Type/LocaleChoiceType.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * CoreShop.
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2015-2019 Dominik Pfaffenbauer (https://www.pfaffenbauer.at)
+ * @license    https://www.coreshop.org/license     GNU General Public License version 3 (GPLv3)
+ */
+
+namespace CoreShop\Bundle\LocaleBundle\Form\Type;
+
+use CoreShop\Component\Resource\Repository\RepositoryInterface;
+use Pimcore\Tool;
+use Symfony\Bridge\Doctrine\Form\DataTransformer\CollectionToArrayTransformer;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class LocaleChoiceType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        if ($options['multiple']) {
+            $builder->addModelTransformer(new CollectionToArrayTransformer());
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver
+            ->setDefaults([
+                'choices' => function (Options $options) {
+                    $locales = Tool::getValidLanguages();
+
+                    /*
+                     * PHP 5.* bug, fixed in PHP 7: https://bugs.php.net/bug.php?id=50688
+                     * "usort(): Array was modified by the user comparison function"
+                     */
+                    @usort($locales, function ($a, $b) {
```

Comment:
Still required?

Reply:
not really, but we should fix that in a separate PR (https://github.com/coreshop/CoreShop/issues/840)